aea/cli/common.py

```diff
@@ -191,20 +190,18 @@ def try_to_load_agent_config(ctx: Context, is_exit_on_except: bool = True) -> No
             logging.config.dictConfig(ctx.agent_config.logging_config)
     except FileNotFoundError:
         if is_exit_on_except:
-            logger.error(
+            raise click.ClickException(
                 "Agent configuration file '{}' not found in the current directory.".format(
                     DEFAULT_AEA_CONFIG_FILE
                 )
             )
```

Comment:
I think this is a good use case for:
```
raise ExceptionX(...) from e
```

see https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement

Reply:
why don't use `AEAException` which can then be re-raised later as a `ClickException`? So we can  distinguish between exception for our logic and exceptions outside it